src/mapml/handlers/ContextMenu.js

```diff
@@ -40,52 +40,23 @@ export var ContextMenu = L.Handler.extend({
         popup:true,
         submenu:[
           {
-            text:"tile",
-            callback:this._copyTile,
+            text:"Map",
+            callback:this._copyMapML,
           },
           {
-            text:"tilematrix",
-            callback:this._copyTileMatrix,
+            text:"Extent",
+            callback:this._copyExtent,
           },
           {
-            spacer:"-",
-          },
-          {
-            text:"map",
-            callback:this._copyMap,
-          },
-          {
-            spacer:"-",
-          },
-          {
-            text:"tcrs",
-            callback:this._copyTCRS,
-          },
-          {
-            text:"pcrs",
-            callback:this._copyPCRS,
-          },
-          {
-            text:"gcrs",
-            callback:this._copyGCRS,
-          },
-          {
-            spacer:"-",
-          },
-          {
-            text: M.options.locale.cmCopyAll,
-            callback:this._copyAllCoords,
+            text:"Location",
```

Comment:
We should probably update the text for "Map", "Extent" and "Location" to be used from the locale (ex. "Map" -> M.options.locale.cmMap) as we are not using acronyms anymore for the copy menu. When you start work on the extension (to add the option to select CS) you can also add new locales for these three text, similar to what's been done on this [PR](https://github.com/Maps4HTML/mapml-extension/pull/44)